android_cobertura.md

```diff
@@ -0,0 +1,143 @@
+# Configure Android Project To Track Code Coverage With Cobertura
+These are the steps required to set up your Jenkins project to output code coverage reports in the Cobertura format. The main argument for doing this is to align both iOS and Android projects and have them all report code coverage via Cobertura instead of separate plugins. **Note:** We are not eliminating JaCoCo from the Android project. We are using JaCoCo to calculate the actual code coverage, but then converting its results into a format that Cobertura can understand for the purposes of reporting the code coverage.
+
+# Jenkins Setup
+
+## coverage.gradle File
+**Note:** This file has been included in the AndroidSkeleton project found [here](https://github.com/IntrepidPursuits/AndroidSkeleton/blob/master/app/coverage.gradle).
+This is the coverage.gradle file used for Northeastern's Android project, so please adapt for your own if they are different. This file needs to be located at `/project-dir/app/coverage.gradle`
```

Comment:
This coverage file can actually be used in any Android project without modification. The only change people might make is if they want to include/exclude additional classes from the coverage.